test_app/lib/screens/join_screen.dart

```diff
@@ -294,15 +321,22 @@ class _JoinConferenceContentState extends State<JoinConferenceContent> {
     var params = ConferenceCreateParameters();
     params.dolbyVoice = switchDolbyVoice;
     params.liveRecording = true;
-    var createOptions = ConferenceCreateOption(conferenceName, params, 0);
+    var createOptions = ConferenceCreateOption(
+        conferenceName, params, 0, SpatialAudioStyle.disabled);
+    if (spatialAudio == true && spatialAudioStyleText == 'Spatial Audio with Individual') {
```

Comment:
Perhaps this `spatialAudioStyleText == 'Spatial Audio with Individual'` is better placed in on change, and perhaps have a constant for the string will be less error prone.